docs/references/phasing.tex

```diff
@@ -1,111 +1,179 @@
-\item{For each time in the \verb!time_array! attribute: Transform the telescope location and the antenna positions or uvws to the ICRS frame (or GCRS if phasing to GCRS).}
-\item{Calculate the relative antenna positions or uvws in the ICRS (or GCRS) frame by subtracting the telescope location in the ICRS (or GCRS) frame.}
-\item{Calculate the phased antenna positions or uvws using the \verb!pyuvdata.utils.phase_uvw! function\footnote{One way to think about what this function does is that is mathematically identical to calculating ENU values, except that it is in the ICRS (or GCRS) frame and the center of the ENU coordinate system is given by the phase center. This makes sense in that if you choose a phase center which is aligned with zenith at the observation time, the changes to the uvws are small and are just due to the changes in the earth spin axis (due to precession and nutation) and the length contraction due to earth's motion around the sun (aberration, excluded if phasing to GCRS).}. If using antenna positions, difference these phased positions to calculate the uvws for each baseline.}
+\item{\texttt{pyerfa}}: A Python-based implementation of the ERFA software package, which is the core library used by \texttt{astropy} in its coordinate transformations. Coordinate rotations are implemented through the method \texttt{erfa.atco13}, and LSTs through \texttt{erfa.gst06a} (the output of which is modified to account for secular polar drift and the observer meridian relative to Greenwich meridian). This implementation is the fastest, although some low-level astrometry effects do not appear to be accounted for, such as Free Core Nutation (FCN; the misalignment between the rotational axis of the Earth's core versus the mantle layer). This can produce errors of order 0.1 mas, which is well below the tolerance required.
+
+\item{\texttt{astropy}}: The \texttt{astropy} software package is one of the core libraries for \texttt{pyuvdata}, which can be used for calculating apparent coordinate positions, and is the sole package used for conversion between different celestial coordinate frames. \texttt{astropy} does not have a frame that corresponds to the topocentric frame per se, but does allow for the calculation of horizontal coordinates (azimith and elevation), which are straightforward to translate to apparent coordinates via matrix rotation knowing the observe latitude and LAST. Like \texttt{pyerfa}, there is no apparent handling of FCN. This implementation is about 50\% slower than \texttt{pyerfa}, although is generally quite consistent with the results of that package.
```

Comment:
Here I think you're using "topocentric" as a synonym for "apparent". Can we stick with "apparent" and note the synonym in the glossary?

Reply:
Yes, I missed this in the text -- I've scrubbed topocentric here, and actually made a note in the coordinate primer at the top.